### ISSUE TITLE
Stored Object: paper:openreview.ry

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed#6939]
{
  "sourceId": "openreview",
  "paperId": "ry",
  "url": "https://openreview.net/forum?id=ry_WPG-A-",
  "title": "On the Information Bottleneck Theory of Deep Learning\ud83d\udcdd",
  "authors": "Andrew Michael Saxe, Yamini Bansal, Joel Dapello, Madhu Advani, Artemy Kolchinsky, Brendan Daniel Tracey, David Daniel Cox",
  "abstract": "The practical successes of deep neural networks have not been matched by theoretical progress that satisfyingly explains their behavior. In this work, we study the information bottleneck (IB) theory of deep learning, which makes three specific claims: first, that deep networks undergo two distinct phases consisting of an initial fitting phase and a subsequent compression phase; second, that the compression phase is causally related to the excellent generalization performance of deep networks; and third, that the compression phase occurs due to the diffusion-like behavior of stochastic gradient descent. Here we show that none of these claims hold true in the general case. Through a combination of analytical results and simulation, we demonstrate that the information plane trajectory is predominantly a function of the neural nonlinearity employed: double-sided saturating nonlinearities like tanh yield a compression phase as neural activations enter the saturation regime, but linear activation functions and single-sided saturating nonlinearities like the widely used ReLU in fact do not. Moreover, we find that there is no evident causal connection between compression and generalization: networks that do not compress are still capable of generalization, and vice versa. Next, we show that the compression phase, when it exists, does not arise from stochasticity in training by demonstrating that we can replicate the IB findings using full batch gradient descent rather than stochastic gradient descent. Finally, we show that when an input domain consists of a subset of task-relevant and task-irrelevant information, hidden representations do compress the task-irrelevant information, although the overall information about the input may monotonically increase with training time, and that this compression happens concurrently with the fitting process rather than during a subsequent compression period.",
  "timestamp": "2025-08-25T07:54:46.294Z",
  "rating": "novote",
  "publishedDate": "",
  "tags": [
    "information bottleneck",
    "deep learning",
    "deep linear networks"
  ],
  "doi": "",
  "journalName": "",
  "sourceType": "url"
}

**Comment by dmarx** [comment]
{
  "_data": {
    "sourceId": "openreview",
    "paperId": "ry",
    "url": "https://openreview.net/forum?id=ry_WPG-A-",
    "title": "On the Information Bottleneck Theory of Deep Learning📝",
    "authors": "Andrew Michael Saxe, Yamini Bansal, Joel Dapello, Madhu Advani, Artemy Kolchinsky, Brendan Daniel Tracey, David Daniel Cox",
    "abstract": "The practical successes of deep neural networks have not been matched by theoretical progress that satisfyingly explains their behavior. In this work, we study the information bottleneck (IB) theory of deep learning, which makes three specific claims: first, that deep networks undergo two distinct phases consisting of an initial fitting phase and a subsequent compression phase; second, that the compression phase is causally related to the excellent generalization performance of deep networks; and third, that the compression phase occurs due to the diffusion-like behavior of stochastic gradient descent. Here we show that none of these claims hold true in the general case. Through a combination of analytical results and simulation, we demonstrate that the information plane trajectory is predominantly a function of the neural nonlinearity employed: double-sided saturating nonlinearities like tanh yield a compression phase as neural activations enter the saturation regime, but linear activation functions and single-sided saturating nonlinearities like the widely used ReLU in fact do not. Moreover, we find that there is no evident causal connection between compression and generalization: networks that do not compress are still capable of generalization, and vice versa. Next, we show that the compression phase, when it exists, does not arise from stochasticity in training by demonstrating that we can replicate the IB findings using full batch gradient descent rather than stochastic gradient descent. Finally, we show that when an input domain consists of a subset of task-relevant and task-irrelevant information, hidden representations do compress the task-irrelevant information, although the overall information about the input may monotonically increase with training time, and that this compression happens concurrently with the fitting process rather than during a subsequent compression period.",
    "timestamp": "2025-08-25T07:54:46.294Z",
    "rating": "novote",
    "publishedDate": "",
    "tags": [
      "information bottleneck",
      "deep learning",
      "deep linear networks"
    ],
    "doi": "",
    "journalName": "",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2025-08-25T07:54:46.977Z",
    "update_mode": "append",
    "issue_number": 6939
  },
  "type": "initial_state"
}
